x-pack/plugin/mapper-unsigned-long/src/main/java/org/elasticsearch/xpack/unsignedlong/UnsignedLongFieldMapper.java

```diff
@@ -170,7 +171,7 @@ public Query termQuery(Object value, SearchExecutionContext context) {
         }
 
         @Override
-        public Query termsQuery(List<?> values, SearchExecutionContext context) {
+        public Query termsQuery(Collection<?> values, SearchExecutionContext context) {
             failIfNotIndexed();
             long[] lvalues = new long[values.size()];
             int upTo = 0;
```

Comment:
The code below tries to access the values by index so the compilation is broken. Can you fix ?

Reply:
@jimczi I'm so sorry for missing a precommit check! It looks good now.